src/locales/index.js

```diff
@@ -1,2 +1,2 @@
-exports.en = { common: require("./en/common.json"), };
-exports.fr = { common: require("./fr/common.json"), };
+exports.en = { common: require("./en/common.json") };
+exports.fr = { common: require("./fr/common.json") };
```

Comment:
this file is generated, we don't want to edit it

Reply:
ah :) let make prettier ignore it